src/NServiceBus.Core/Persistence/PersistenceDefinition.cs

```diff
@@ -68,6 +68,14 @@ public bool HasSupportFor(Storage storage)
             return storageToActionMap.ContainsKey(typeof(T));
         }
 
+        /// <summary>
+        /// True if supplied storage is supported
+        /// </summary>
+        public bool HasSupportFor(Type storageType) 
```

Comment:
The real question is if the generic method should have been public at all. It's only used by a couple of acceptance test AFAICS.